src/server/scripts/Northrend/Ulduar/Ulduar/boss_yoggsaron.cpp

```diff
@@ -296,6 +298,25 @@ enum Misc
     DATA_GET_DRIVE_ME_CRAZY             = 4,
 };
 
+struct LocationsXY
+{
+    float x, y, z, o;
+};
+
+static LocationsXY yoggPortalLoc[] =
```

Comment:
Are we sure there is not already such a structure? If not, we might define a generic one so we can always reuse the same.

Approving the PR already, as the above can be done even later

Reply:
ah, also the `o` is never used here. @Si1ker please remove it.

Reply:
@FrancescoBorzi change applied.